static/src/javascripts/projects/common/modules/identity/ad-prefs.js

```diff
@@ -28,8 +26,14 @@ type ConsentRadioButtonProps = {
 
 type ConsentBoxProps = {
     consent: AdConsent,
-    state: AdConsentState,
-    onUpdate: (state: AdConsentState) => void,
+    state: ?boolean,
+    onUpdate: (state: ?boolean) => void,
+};
+
+type AdPrefsWrapperProps = {
+    getAdConsentState: (consent: AdConsent) => ?boolean,
```

Comment:
Is `?boolean` an Optional? If so I generally think Booleans can have sane defaults

Reply:
What do you mean?